integration/tests/runtime-workflow-postgres.ts

```diff
@@ -586,44 +626,46 @@ test('Delete Note 1', async () => {
 });
 
 test('getComment in simultaneous with different resolvers pinfo', async () => {
-  const { data } = await client.query({ operationName:"getCommentDetailsAtOnce", query: GET_COMMENTS , variables: {
-    id: 1
-  }});
+  const { data } = await client.query({
+    operationName: "getCommentDetailsAtOnce", query: GET_COMMENTS, variables: {
+      id: 1
+    }
+  });
```

Comment:
This is not related, but I am curios what editor settings do you have. Maybe we can share them across. I usually do not care about this sort of things and run the  `"linter"` hoping it will take care of it.